dask_cuda/cli.py

```diff
@@ -87,9 +87,10 @@ def cuda():
     "--memory-limit",
     default="auto",
     show_default=True,
-    help="""Bytes of memory per process that the worker can use. Can be an integer
-    (bytes), float (fraction of total system memory), string (like ``"5GB"`` or
-    ``"5000M"``), or ``"auto"`` or 0 for no memory management.""",
+    help="""Size of the host LRU cache, which is used to determine when the worker
+    starts spilling to disk (not available if JIT-Unspill is enabled). Can be an
```

Comment:
is there any dependency between memory-limit and device-memory-limit, that should be set? like if one cannot be bigger than other etc..

Reply:
`memory-limit` and `device-memory-limit` are independent